client/src/legacy/GridField.js

```diff
@@ -64,7 +64,13 @@ $.entwine('ss', function($) {
       // Include any GET parameters from the current URL, as the view state might depend on it.
       // For example, a list prefiltered through external search criteria might be passed to GridField.
       if(window.location.search) {
-        ajaxOpts.data = window.location.search.replace(/^\?/, '') + '&' + $.param(ajaxOpts.data);
+        let searchParams = window.location.search.replace('?', '').split('&');
```

Comment:
I think we need to keep the `/^\?/` regex because URLs can contain `?` that aren't at the beginning

Reply:
Added in commit #2a3bf4c